server/odc-service/src/main/java/com/oceanbase/odc/service/task/util/JobUtils.java

```diff
@@ -180,4 +180,12 @@ public static void putEnvToSysProperties(String environmentKey) {
     public static void encryptEnvironments(Map<String, String> environments) {
         new JobEnvironmentEncryptor().encrypt(environments);
     }
+
+    public static String encrypt(String key, String salt, String raw) {
```

Comment:
JobEnvironmentEncryptor means encrypt. but is contains decrypt method. 
if we don't implement JobEnvironmentDecryptor,  
move  encrypt and decrypt method from JobEnvironmentEncryptor. to here is a better choice

Reply:
Yes, it's wired. But before I added this `decrypt` method, it has already existed a method `decrypt`, so I followed this. And you could also see `TextEncryptor`